prepare/formats/human_assistant.py

```diff
@@ -1,8 +1,9 @@
 from src.unitxt.catalog import add_to_catalog
-from src.unitxt.formats import ICLFormat
+from src.unitxt.operators import ModelInputFormatter
 
-format = ICLFormat(
-    input_prefix="Human: ", output_prefix="Assistant: ", input_output_separator="\n"
+format = ModelInputFormatter(
+    demo_format="Human: {source}\nAssistant: {target}\n\n",
+    model_input_format="{instruction}\n{demos}\nHuman: {source}\nAssistant: ",
```

Comment:
Same issue as above.  Remove \n after {demos}

Reply:
removed the \n after demos. But this is a different issue. it does not relate to the instruction.